misc/scripts/deploy-tools.sh

```diff
@@ -222,7 +222,7 @@ function cleanvm {
 function cleanupVMPre {
     case $FILE_EXT in
     deb)
-        $SSH $TARGET "$DEB_QUERY $PLUGIN_NAME > /dev/null"
+        $SSH $TARGET "$DEB_QUERY $PLUGIN_NAME > /dev/null 2>&1"
```

Comment:
Here is my understanding:

CI master run (passed and no indication of race): https://ci.vmware.run/vmware/docker-volume-vsphere/936

`dpkg-query -l docker-volume-vsphere`  responds back with following error message if the docker-volume-vsphere (deb package) is not present

```
=> cleanvm: Cleaning up on root@192.168.31.83 : deb Thu Jul 6 20:29:03 UTC 2017

dpkg-query: no packages found matching docker-volume-vsphere
```

The above message comes always even while running locally against ubuntu vm. We haven't remove deb package hence cleanVM routine is still having this query check first. I would say it is harmless.